armi/reactor/converters/uniformMesh.py

```diff
@@ -123,15 +123,29 @@ def generateCommonMesh(self):
             :id: I_ARMI_UMC_NON_UNIFORM
             :implements: R_ARMI_UMC_NON_UNIFORM
 
+            Generates a core-wide mesh that preserves important material boundaries, in
+            particular the terminations of the fuel column as well as control assembly
+            material. Calls ``_computeAverageAxialMesh``, which operates by first
+            collecting all the mesh points for every assembly (``allMeshes``) and then
+            averaging them together using ``average1DWithinTolerance`` to obtain the
+            core-wide mesh.
+
```

Comment:
```suggestion
            The preservation of important material boundaries is only enabled if the `minimumMeshSize` is defined. This is controlled by the `uniformMeshMinimumSize` setting.
```

Reply:
Need to wrap this text but it's hard to do in the browser interface

Reply:
https://github.com/terrapower/armi/pull/1618/commits/b8279eecfde888ed4eb622bdd24fe2879ed2021d try this